src/routes/blog/posts/v3-2/+page.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Announcing Files v3.2
+description: List View layout, edit album covers, higher quality thumbnails.
+thumbnail: /blog-resources/v3-2/Hero.png
+date: 1/8/2024
+author: files-community
+---
+
+We're excited to announce that starting today, Files v3.2 is now available. If you already have Files, there should be a notification on the toolbar to install the update. Otherwise, you can download Files for free from our [download page](/download/). You can also purchase Files on the [Microsoft Store](ms-windows-store://pdp/?ProductId=9nghp3dx8hdx&cid=FilesWebsite) or sponsor us on [GitHub](https://github.com/sponsors/yaira2) to help support the project.
+
+**TL;DR:** Files v3.2 adds a List View layout, support for editing album covers from the properties window, higher quality thumbnails, and more. Continue reading to learn more about these changes.
+
+## What's new in v3.2
+
+### Browse large folders with confidence
+
+We know how frustrating it can be when Files crashes or freezes while browsing large folders. That’s why we’ve made significant improvements to enhance stability. Our team has worked hard to reduce crashes and improve performance when browsing large folders and when adding and deleting items. We believe these improvements will make your experience smoother and more enjoyable.
```

Comment:
```suggestion
We know how frustrating it can be when Files crashes or freezes while browsing large folders. That’s why we’ve made significant improvements to enhance stability. Our team has worked hard to reduce crashes when browsing large folders and when adding and deleting items. We believe these improvements will make your experience smoother and more enjoyable.
```